layers/state_tracker/pipeline_state.h

```diff
@@ -132,8 +132,15 @@ class Pipeline : public StateObject {
 
     mutable bool binary_data_released = false;
 
-    // We create a VkShaderModule that is instrumented and need to delete before leaving the pipeline call
-    std::vector<VkShaderModule> instrumented_shader_module;
+    // TODO - Because we have hack to create a pipeline at PreCallValidate time (for GPL) we have no proper way to create inherited
+    // state objects of the pipeline This is to make it clear that while currently everyone has to allocate this memory, it is only
+    // ment for GPU-AV
+    struct GPU {
```

Comment:
Since it only contains instrumentation data, let's be more specific
```suggestion
    struct InstrumentationData {
```